convert.py

```diff
@@ -39,21 +40,31 @@ def convert(flags):
         groups.append(group)
 
     manifest = generate_manifest(flags.survey_title, groups)
-    with open(flags.manifest_out, 'w') as f:
+    manifest_file = os.path.join(flags.manifest_out, flags.survey_title + '.yaml')
+    with open(manifest_file, 'w') as f:
         yaml.dump(manifest, f, default_flow_style=False)
 
 
-def create_yaml_blocks(flags, block):
-    try:
-        with open('{}/{}.yaml'.format(flags.blocks_out, block['id']), 'r+') as f:
+def create_yaml_block(flags, block):
+    """
+    Checks whether block name exists, and if content matches that of an existing block
```

Comment:
not quite what this function is doing though...

`Checks if a YAML file already exists for the given block, if so, compares the content of the file with the given block and creates a new variant YAML file if different. If there is no existing YAML file then a new one is created.`